scaleway/provider.go

```diff
@@ -26,6 +26,11 @@ func DefaultProviderConfig() *ProviderConfig {
 
 // Provider returns a terraform.ResourceProvider.
 func Provider(config *ProviderConfig) plugin.ProviderFunc {
+	// Catch every panic after this line. This will send an anonymous report on Scaleway's sentry.
```

Comment:
This should be done in the main as this function immediately exist so no error will be panic in this context